client/components/Board.js

```diff
@@ -14,17 +14,22 @@ class Board extends Component {
   }
 
   render () {
+    var counter = 0
+    var levelTheme = this.props.currentLevel < 5 ? 'basement' : 'cargo'
+    console.log('the level ', levelTheme)
     return (
       <div id='board'>
         <div className='board-container'>
           {
             this.props.board.map((row, rowIndex) => {
               return row.map((col, colIndex) => {
+                let oddEven = counter % 2 == 1 ? 'odd' : 'even'
```

Comment:
You don't need a counter for alternating doing alternating in a 2D array.
`let oddEven = rowIndex + colIndex % 2 == 1 ? 'odd' : 'even'`
Achieves the same result.